_posts/2021-12-10-start-a-workspace-by-entering-a-url.adoc

```diff
@@ -0,0 +1,98 @@
+---
+title: Start a workspace by entering a URL
+layout: post
+author: Florent Benoit
+description: >-
+  How to start a workspace by customizing the URL.
+categories: []
+keywords: ['GitHub', 'GitLab', 'Bitbucket', 'start', 'workspace']
+slug: /@florent.benoit/start-a-workspace-by-entering-a-url
+---
+
+== Starting a workspace by entering a URL
+
+As a user, you can create a workspace containing a clone of a Git repository by entering a custom URL in the address bar of your browser.
+
+As a user, you can also create a workspace containing a clone of a Git repository by entering the URL of the Git repository on the {prod} dashboard page.
+
+=== Short URL syntax for cloning hosted Git repositories
+
+In the earlier versions of {prod}, the {prod-short} URL syntax began with `/f/?url=`, for example `{prod-url}/f/?url=https://github.com/my/repository`, but it was complex and error-prone. While that syntax is still supported, short syntax should be used as follows:
+
+To start a workspace with a clone of a GitHub, GitLab, or Bitbucket repository, go to your browser's address bar and enter the {prod-short} URL, prepended with a `#` character and appended with the repository link. Using the example of Eclipse Che hosted by Red Hat: link:https://workspaces.openshift.com#https://github.com/eclipse-che/che-server[]. Then {prod} starts a workspace and clones the repository, `\https://github.com/eclipse-che/che-server` in this example.
+
+
+{prod} also searches for a `.devfile.yaml` or `devfile.yaml` file in the given repository. The devfile describes the containers and commands to use for that project. Omitting this file will result in a default container, which provides different languages (including Java, Node.js, Python, PHP, Golang, etc) and tools (including `curl`, `jq`, `yq`, etc).
```

Comment:
I didn't know about starting a  workspace without a devfile. That's great!

Reply:
this is probably smth. that will be available starting from 7.41.0